.github/workflows/build.yml

```diff
@@ -11,12 +11,12 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v2
               with:
-                  node-version: '14'
+                  node-version: '16.10.0'
                   cache: 'npm'
                   registry-url: 'https://registry.npmjs.org'
 
             - name: Install dependencies
-              run: npm install
+              run: npm install --legacy-peer-deps
```

Comment:
did you hvae to run with the --legacy-peer-deps locally? I added this early on, but I wonder if it's actually needed now? It'd be ideal if we can remove it.

Reply:
I did use the run: npm install --legacy-peer-deps to install the npm packages, we might need that as executing just "npm i" doesn't really install successfully.

that's what i see when ran "npm install"
![image](https://github.com/microsoft/iot-cardboard-js/assets/138707647/cf481bb6-536d-4320-8247-a1d68a86b7b1)